Packs/Slack/Integrations/SlackV3/README.md

```diff
@@ -29,6 +29,8 @@ Please refer to the video tutorial [found here](https://live.paloaltonetworks.co
     | `filtered_tags` | Comma-separated list of tags by which to filter the messages sent from XSOAR. Only supported in Cortex XSOAR V6.1 and above. | False |
     | `permitted_notifications` | Types of Notifications to send in the dedicated channel. | False |
     | `channel_fetch_interval` | The frequency with which to update the channel IDs. **Note:** For workspaces that contain large amounts of channels, it is advised to periodically fetch the new channels. | False |
+    | `users_fetch_interval` | The frequency with which to update the user IDs. **Note:** For workspaces that contain large amounts of users, it is advised to periodically fetch the new users. | False |
+
```

Comment:
I'm not sure. On the one hand, it would allow us to reduce the amount of params, but on the other hand it can allow customers to configure each independently.